internal/resource.go

```diff
@@ -90,17 +104,7 @@ func (l *Resource) Download(dir string, mode os.FileMode, ctx context.Context) e
 	}
 	var downloadError error = nil
 	for _, u := range l.Urls {
-		// Download file in the target directory so that the call to
```

Comment:
This comment needs to stay, it's quite important to explain why we're doing this.

Reply:
I don't see the comment re-instated.